app/src/main/res/drawable/ic_add_donation.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#8CB4B0"
```

Comment:
Are those files still necessary? Please, make sure and delete if they are not :)

Reply:
No, they are not necessary anymore. To be removed.